alignment/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-bin.zip
```

Comment:
Any reason why using 4.10 instead of current 5.x version? ( https://gradle.org/install/ )

Reply:
Note that this is just the beginning of the work… :)

Reply:
Yeah totally :-)

Reply:
Updated.